scripts/gateway.service

```diff
@@ -0,0 +1,39 @@
+# This is a SystemD unit file for the Gateway
+# To configure:
+# - Update the user and group that the service will run as (User & Group below) or create one with:
+#   'sudo adduser --system --no-create-home --group --disabled-login storj'
+# - Ensure that the gateway binary is in /usr/bin and is named gateway (or edit the ExecStart line
+#   below to reflect the name and location of your binary
+# - Ensure that you've run setup and have edited the configuration appropriately prior to starting the
+#   service with this script and adjust the --config-dir option in the ExecSart line if necessary
+#   'sudo gateway setup --config-dir /etc/storj/gateway/'
```

Comment:
maybe run this after the chown but with `sudo -u storj`?

Reply:
Yeah that would work as well!

Reply:
we'll probably need a mkdir in there then